src/component/admin/groups/CreateGroup/CreateGroup.tsx

```diff
@@ -62,6 +67,17 @@ export const CreateGroup = () => {
         navigate(GO_BACK);
     };
 
+    const onSetName = (name: string) => {
+        clearErrors();
+        if (groups?.filter(group => group.name === name).length) {
```

Comment:
Could extract to a variable for clarity. Now you need to read and understand the expression to know what happens here. Makes it hard to scan the code.

Reply:
Addressed in https://github.com/Unleash/unleash-frontend/pull/1210/commits/84c6a37dcbe94b9eb34951200484f07fa3345cd5